src/api/privateApi.js

```diff
@@ -299,12 +262,6 @@ function handleSendRequestFailures(requestMethod, requestType, options, error) {
 	});
 }
 
-function optionallyCallCallback(callback, result) {
-	if (typeof callback === 'function') {
-		callback(result);
-	}
-	return result;
-}
 
 module.exports = {
```

Comment:
Functions should also be exported from the utils file instead of this one.

Reply:
That's right, fixed that in the new commit, and updated tests